deploy/crontab-test-apps

```diff
@@ -23,9 +23,9 @@
-*/30	*	* 	* 	*	export NOW=`date +\%F_\%H\%M\%S` ; export DOAJENV=test ; echo -e "\n\nstart timestamp $NOW" >> /home/cloo/cron-logs/doaj-ingest-articles.log 2>&1 && /home/cloo/repl/apps/doaj/bin/python /home/cloo/repl/apps/doaj/src/doaj/portality/scripts/ingestarticles.py >> /home/cloo/cron-logs/doaj-ingest-articles.log 2>&1
-*/30	*	* 	* 	*	export NOW=`date +\%F_\%H\%M\%S` ; export DOAJENV=test ; echo -e "\n\nstart timestamp $NOW" >> /home/cloo/cron-logs/doaj-news-fetch.log 2>&1 && /home/cloo/repl/apps/doaj/bin/python /home/cloo/repl/apps/doaj/src/doaj/portality/scripts/news.py >> /home/cloo/cron-logs/doaj-news-fetch.log 2>&1
-*/30	*	* 	*	*	export NOW=`date +\%F_\%H\%M\%S` ; export DOAJENV=test ; echo -e "\n\nstart timestamp $NOW" >> /home/cloo/cron-logs/doaj-journal-csv.log 2>&1 && /home/cloo/repl/apps/doaj/bin/python /home/cloo/repl/apps/doaj/src/doaj/portality/scripts/journalcsv.py >> /home/cloo/cron-logs/doaj-journal-csv.log 2>&1
-*/10    *       *       *       *       export NOW=`date +\%F_\%H\%M\%S` ; export DOAJENV=test ; echo -e "\n\nstart timestamp $NOW" >> /home/cloo/cron-logs/doaj-ingest-reapps.log && /home/cloo/repl/apps/doaj/bin/python /home/cloo/repl/apps/doaj/src/doaj/portality/scripts/ingestreapplications.py >> /home/cloo/cron-logs/doaj-ingest-reapps.log 2>&1
-
+# m 	h   dom mon dow command
+*/30    *   * 	* 	*   export NOW=`date +\%F_\%H\%M\%S` ; export DOAJENV=test ; echo -e "\n\nstart timestamp $NOW" >> /home/cloo/cron-logs/doaj-ingest-articles.log 2>&1 && /home/cloo/repl/apps/doaj/bin/python /home/cloo/repl/apps/doaj/src/doaj/portality/scripts/ingestarticles.py >> /home/cloo/cron-logs/doaj-ingest-articles.log 2>&1
+*/30    *	* 	* 	*   export NOW=`date +\%F_\%H\%M\%S` ; export DOAJENV=test ; echo -e "\n\nstart timestamp $NOW" >> /home/cloo/cron-logs/doaj-news-fetch.log 2>&1 && /home/cloo/repl/apps/doaj/bin/python /home/cloo/repl/apps/doaj/src/doaj/portality/scripts/news.py >> /home/cloo/cron-logs/doaj-news-fetch.log 2>&1
+*/30    *	* 	*	*   export NOW=`date +\%F_\%H\%M\%S` ; export DOAJENV=test ; echo -e "\n\nstart timestamp $NOW" >> /home/cloo/cron-logs/doaj-journal-csv.log 2>&1 && /home/cloo/repl/apps/doaj/bin/python /home/cloo/repl/apps/doaj/src/doaj/portality/scripts/journalcsv.py >> /home/cloo/cron-logs/doaj-journal-csv.log 2>&1
+*/10    *   *   *   *   export NOW=`date +\%F_\%H\%M\%S` ; export DOAJENV=test ; echo -e "\n\nstart timestamp $NOW" >> /home/cloo/cron-logs/doaj-ingest-reapps.log && /home/cloo/repl/apps/doaj/bin/python /home/cloo/repl/apps/doaj/src/doaj/portality/scripts/ingestreapplications.py >> /home/cloo/cron-logs/doaj-ingest-reapps.log 2>&1
+    0   7   *   *   1   export NOW=`date +\%F_\%H\%M\%S` ; export DOAJENV=test ; echo -e "\n\nstart timestamp $NOW" >> /home/cloo/cron-logs/doaj-workflow-emails.log && /home/cloo/repl/apps/doaj/bin/python /home/cloo/repl/apps/doaj/src/doaj/portality/scripts/async_workflow_notifications.py >> /home/cloo/cron-logs/doaj-workflow-emails.log 2>&1
```

Comment:
this seems a bit wrong. Should it be 

```0   7   *   *   1   *```

instead of 

```spacespacespacespace0   7   *   *   1``` (additional asterisk at the end for the day of week (dow) column, no space at the beginning of cron spec)